src/common/util/copy-clipboard.ts

```diff
@@ -1,8 +1,12 @@
 export const copyToClipboard = (str) => {
-  const el = document.createElement("textarea");
-  el.value = str;
-  document.body.appendChild(el);
-  el.select();
-  document.execCommand("copy");
-  document.body.removeChild(el);
+  if (navigator.clipboard) {
+    navigator.clipboard.writeText(str);
```

Comment:
This will require permission, that is also not supported in all browsers (Firefox, Safari) we should check if this is successful and otherwise fallback

Reply:
It shouldn't require permission as long as it's user-initiated. It works just fine for me in the latest version of Firefox, don't have a Mac so can't test safari. Do you mean that the `if (navigator.clipboard) {` might not catch it? I put a fallback in anyway.